<launch>
    <arg name="upper_arm_length" default="0.25" />
    <arg name="forearm_length" default="0.2" />
    <arg name="hand_length" default="0.1" />
    <arg name="head_radius" default="0.1" />
    <arg name="torso_length" default="0.6" />
    <arg name="torso_radius" default="0.15" />
    <arg name="waist_radius" default="0.12" />
    <arg name="waist_length" default="0.2" />
    <arg name="neck_length" default="0.1" />
    <arg name="thigh_length" default="0.4" />
    <arg name="shin_length" default="0.35" />
    <arg name="foot_length" default="0.1" />
    <arg name="shoulder_offset_width" default="0.15" />
    <arg name="shoulder_offset_height" default="0.3" />
    <arg name="hip_offset_width" default="0.1" />
    <arg name="hip_offset_height" default="-0.05" />
    
    <param name="human_description" command="$(find xacro)/xacro.py $(find kinect_skeleton_publisher)/urdf/human_skeleton.urdf.xacro upper_arm_length:=$(arg upper_arm_length) forearm_length:=$(arg forearm_length) hand_length:=$(arg hand_length) head_radius:=$(arg head_radius) torso_length:=$(arg torso_length) torso_radius:=$(arg torso_radius) waist_radius:=$(arg waist_radius) waist_length:=$(arg waist_length) neck_length:=$(arg neck_length) thigh_length:=$(arg thigh_length) shin_length:=$(arg shin_length) foot_length:=$(arg foot_length) shoulder_offset_width:=$(arg shoulder_offset_width) shoulder_offset_height:=$(arg shoulder_offset_height) hip_offset_width:=$(arg hip_offset_width) hip_offset_height:=$(arg hip_offset_height)" />
</launch>
